README.md

```diff
@@ -43,6 +43,11 @@ Requirements:
 - [Docker](https://www.docker.com/get-docker)
 - [Docker-compose](https://docs.docker.com/compose/)
 
+Copy `docker-compose.yml.dist` into `docker-compose.yml`.
+
+You can make any alterations you need for your local Docker setup but the defaults
+should be enough to set the site up.
```

Comment:
Instead of the 'site', I would suggest using the word 'project' as it might give a wrong impression to some users. Also, I think that in this case there should be a colon before the but because of connecting two independent clauses ;)